### PR TITLE
xlisp #106: allow 32-bit builds to fail

### PR DESCRIPTION
**Pull request by wobh**
- copy and use local install script
- make install script verbose
- add arch check, abcl
- use proper key in allow_failures

### CODE REVIEW

**Comment by wobh** [comment]
This brings in the installation script from `https://github.com/luismbo/cl-travis/raw/master/install.sh` which I'm fine with for now, even though, I kinda think it means we have to fix it. We can always set the travis config back if a real improvement happens upstream.

**Comment by wobh** [comment]
PR for #104 to follow this one.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Exercism exercises in Lisp
 
-[![Build Status](https://travis-ci.org/exercism/xlisp.png)](https://travis-ci.org/exercism/xlisp)
+[![Build Status](https://travis-ci.org/wobh/xlisp.png)](https://travis-ci.org/wobh/xlisp)
```
Comment (verdammelt) [inline README.md]: @wobh could you please change this back? Other than that - merge this PR.

Comment (wobh) [inline README.md]: Doh! Thank you!

**Pull request merged**